yeast-markdown-parser/src/plugins/block/TableParser.ts

```diff
@@ -178,3 +166,19 @@ const abbreviateAlignment = (alignments: string[]) => {
 		})
 		.join('|');
 };
+
+function normalizePipeString(input: string): string {
```

Comment:
@Zino-ctrlZ can you add a comment to this function to explain it what way it's being normalized? I think it's making sure that unescaped pipes are surrounded by whitespace so they're not touching other characters, but it would be a lot easier to read a comment saying that in the future. :)